stdlib/universe/typeconv_test.go

```diff
@@ -5,32 +5,239 @@ import (
 	"github.com/influxdata/flux/values"
 )
 
+func TestTypeconv_String(t *testing.T) {
+	testCases := []struct {
+		name  	  string
+		v     	  interface{}
+		want  	  string
+		expectErr error
+	} { 
+		{
+			name : "string(v:1)",
+			v : int64(541),
+			want : "541",
+		},
+		{
+			name : "string(v:2)",
+			v : uint64(501),
+			want : "501",
+		},
+		{
+			name : "string(v:3)",
+			v : float64(653.28),
+			want : "653.28",
+		},
+		{
+			name : "string(v:4)",
+			v : bool(true),
+			want : "true",
+		},
+		{
+			name : "string(v:5)",
+			v : bool(false),
+			want : "false",
+		},
+		{
+			name : "string(v:6)",
+			v : values.Time(1136239445999999999),
+			want : "2006-01-02T22:04:05.999999999Z",
+		},
+		{
+			name : "string(v:7)",
+			v : values.Duration(184000000000),
+			want : "3m4s",
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t*testing.T) {
+			myMap := map[string]values.Value {
+				"v" : values.New(tc.v),
+			}
+			args := values.NewObjectWithValues(myMap)
+			c := &stringConv{}
+			got, expectErr := c.Call(args)
+			if expectErr != nil {
+				t.Error(expectErr)
+			}
+			want := values.NewString(tc.want)
+			if !got.Equal(want) {
+				t.Errorf("Wanted: %s, got: %v", want, got)
+			}
+		})
+	}
+}
+
+func TestTypeconv_Int(t *testing.T) {
+	testCases := []struct {
+		name  	  string
+		v     	  interface{}
+		want  	  int64
+		expectErr error
+	} { 
+		{
+			name : "int64(v:1)",
+			v : "4615",
+			want : int64(4615),
+		},
+		{
+			name : "int64(v:2)",
+			v : uint64(123),
+			want : int64(123),
+		},
+		{
+			name : "int64(v:3)",
+			v : float64(728),
+			want : int64(728),
+		},
+		{
+			name : "int64(v:4)",
+			v : true,
+			want : int64(1),
+		},
+		{
+			name : "int64(v:4)",
+			v : false,
+			want : int64(0),
+		},
+		{
+			name : "int64(v:5)",
+			v : values.Time(1136239445999999999),
+			want : int64(1136239445999999999),
+		},
+		{
+			name : "int64(v:6)",
```

Comment:
we do in fact want to verify that functions return errors as expected, so we want a case like this: 
```
{
  name: "cannotConvertInt", 
  v: "notanumber"
  want: nil
  wantErr: errors.new("cannot parse notanumber as an integer")
}
```

and then in your test case driver, 

```
result, err := c.Call(args)
if tc.wantErr != nil {
	if got, want := err.Error(), tc.wantErr.Error(); got != want {
		t.Errorf("unexpected error want: %s got: %s", want, got)
 	}
	return
}
```